### PR TITLE
remove extraneous session_start()

### PR DESCRIPTION
**Pull request by kingfui**

### CODE REVIEW

**Comment by zelazowy** [comment]
up!

**Comment by zelazowy** [comment]
Guys, can you merge this PR? With this extraneous `session_start()` calls it's not working and I have to create my own form just for removing them. It's nearly 3 months...

File: src/Evernote/Auth/OauthHandler.php [hunk 1]
```diff
@@ -116,7 +114,8 @@ protected function getTemporaryCredentials()
             CURLOPT_HTTPHEADER     => $this->formatHeaders($headers),
             CURLOPT_POSTFIELDS     => http_build_query($arguments, '', '&'),
             CURLOPT_HEADER         => true,
-            CURLOPT_RETURNTRANSFER => true
+            CURLOPT_RETURNTRANSFER => true,
+			CURLOPT_SSL_VERIFYPEER => false
```
Comment (akhaku) [inline src/Evernote/Auth/OauthHandler.php]: hm, why the change here?

Comment (kingfui) [inline src/Evernote/Auth/OauthHandler.php]: Sorry, change is irrelevant to the fix. Accidentally pushed. Will look into removing the commit now.
